foyer-storage/src/region.rs

```diff
@@ -88,16 +88,21 @@ where
         &self.stats
     }
 
-    pub async fn write(&self, buf: IoBuffer, offset: u64) -> Result<()> {
-        self.device.write(buf, self.id, offset).await
+    pub async fn write(&self, buf: IoBuffer, offset: u64, flush: bool) -> Result<()> {
+        self.device.write(buf, self.id, offset).await?;
+        if flush {
+            self.flush().await?;
+        }
+
+        Ok(())
     }
```

Comment:
The read/write/flush interfaces with `Region` are supposed to be a proxy of the interfaces of the `Device`. `flush` here implies a `sync(2)` command under the hood, which is not optimized for performance.

Reply:
Thanks for the answer, I'll close this PR for now